w3d_translator/w3d_translator.py

```diff
@@ -10,12 +9,14 @@
 
 UNITY_VERSION = "2021.3.8f1"
 UNITY_PATH = Path(
-    f"C:\\Program Files\\Unity\\Hub\\Editor\\{UNITY_VERSION}\\Editor\\Unity.exe" # noqa
+    f"C:\\Program Files\\Unity\\Hub\\Editor\\{UNITY_VERSION}\\Editor\\Unity.exe"  # noqa
 )
 STARTER_PROJECT = Path("unity/CAVE")
 LOG_FLAG = "LOG:"  # Flag to send prints from the CLI script onto the console
 console = Console()
-err_console = Console(stderr=True)
+err_console = Console(stderr=True, style="bold red")
```

Comment:
Is this to log the events in the python script?  If that's the case, why not use [logging](https://docs.python.org/3/library/logging.html)

Reply:
I've been liking [rich](https://rich.readthedocs.io/en/stable/introduction.html) for it's various, some of which I think will come in really handy once we start multithreading. I'll definitely take a look at logging though! We may be able to use both